packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -344,9 +385,14 @@ class ReactSixteenAdapter extends EnzymeAdapter {
     };
   }
 
-  createShallowRenderer(/* options */) {
+  createShallowRenderer(options = {}) {
     const adapter = this;
     const renderer = new ShallowRenderer();
+    const { suspenseFallback } = options;
+    if (typeof suspenseFallback !== 'undefined' && typeof suspenseFallback !== 'boolean') {
+      throw TypeError('`options.suspenseFallback` should be boolean or undefined');
+    }
+    const shouldReplaceLazyWithFallback = typeof suspenseFallback === 'undefined' ? true : !!suspenseFallback;
```

Comment:
```suggestion
    const shouldReplaceLazyWithFallback = typeof suspenseFallback === 'boolean' ? suspenseFallback : true;
```

altho i wonder if maybe we could invert this, so that "absent" and `false` are the same?

Reply:
That makes sense since now we throw error when it's not boolean or undefined.